defining own regions extend (#4429)
vcmi/vcmi: docs/modders/Campaign_Format.md

```js
{

>     "background": "own_region_background.bmp",

Reviewer: - use camelCase
- suggest to use png instead of bmp's
```suggestion
    "background": "ownRegionBackground.png",
```

		"suffix": ["Enabled", "Selected", "Conquered"],
    "prefix": "G3",

>     "color_suffix_length": 1,

Reviewer: I know this is not yours code, but can we change it to `colorSuffixLength` before we have campaigns we need to support?
 We always use camelCase in json's

},
```

- `"background"` optional - use own image name for background instead of adding "_BG" to the prefix as name
- `"prefix"` used to identify all images related to campaign. In this example (if background parameter wouldn't exists), background picture will be `G3_BG`
- `"suffix"` optional - use other suffixes than the default `En`, `Se` and `Co` for the three different images

> - `"infix"` ised to identify all images related to region. In this example, it will be pictures starting from `G3A_..., G3B_..., G3C_..."` 

Reviewer 2: ```suggestion
- `"infix"` used to identify all images related to region. In this example, it will be pictures whose files names begin with `G3A_..., G3B_..., G3C_..."` 
```
